CONTRIBUTING.md

```diff
@@ -13,6 +13,45 @@ sure you have those two installed and then run tests against all supported Cloju
 
     lein all test
 
+
+#### Testing with docker
+
+* pull [Elasticsearch]() image 
```

Comment:
This links to nothing. I'm happy to merge this without fixing this but it should be fixed at a later point.